extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -808,7 +808,7 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
         }
         displayEmail = '<div class="select_contact_email" data-test="action-select-contact-email">' + Xss.escape(displayEmail) + '</div>';
         if (contact.name) {
-          ulHtml += '<div class="select_contact_name" data-test="action-select-contact-name">' + Xss.escape(contact.name) + displayEmail + '</div>';
+          ulHtml += '<div class="select_contact_name" data-test="action-select-contact-name">' + Xss.stripEmojis(contact.name) + displayEmail + '</div>';
```

Comment:
no need for `stripEmojis` here, let's change it back to `escape`

Reply:
ok - will do. thanks.